### DISCUSSION TITLE
Update wiki for NES Headers

### DISCUSSION THREAD

**Discussion by kdawg8** [sanni/cartreader#172] [General Chat]
You're "Dumping NES Games" wiki page says that NES games dump without the header, so I was pleasantly surprised when my new cart reader added the header to recognized carts.  Can you update the wiki page?

**Comment by kdawg8** [comment]
Thanks for updating the Wiki!

For anyone else dumping NES ROMs, the nes20db header database is pretty incomplete as far as licensed titles go.  I've dumped about 20 of my games so far and only about half of them got headers added.  The database is mostly homebrews, with some of the more popular licensed games included.

An alternative way to add the iNES header if you're not familiar with how to run Python code (like myself) is to use a hex editor such as HxD to insert 16 Bytes into the beginning of the CART.bin file (increasing the file size), look up the header from http://nes.dnsabr.com/, and paste it over the 16 new Bytes you added.  Change the file type to .nes and it will run in an emulator.

**Comment by sanni** [comment]
Cool site, thanks for sharing 👍
